app/scripts/modules/azure/instance/details/instanceDetails.html

```diff
@@ -0,0 +1,221 @@
+<div class="details-panel">
+  <div class="header" ng-if="state.loading">
+    <div class="close-button" ng-if="!state.standalone">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove"></span>
+      </a>
+    </div>
+    <h4 class="text-center">
+      <span us-spinner="{radius:20, width:6, length: 12}"></span>
+    </h4>
+  </div>
+
+  <div class="header" ng-if="!state.loading">
+    <div class="close-button" ng-if="!state.standalone">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove"></span>
+      </a>
+    </div>
+    <div class="header-text">
+      <span class="glyphicon glyphicon-hdd {{instance.healthState}}"></span>
+      <h3 select-on-dbl-click>
+        {{instance.instanceId}}
+      </h3>
+
+      <copy-to-clipboard
+          ng-if="instance.provider === 'azure'"
+          class="copy-to-clipboard"
+          text="ssh -t azure.{{instance.account}}.netflix.net 'oq-ssh --region {{instance.region}} {{instance.instanceId}}'"
+          tool-tip="Copy SSH command to clipboard">
+      </copy-to-clipboard>
```

Comment:
Remove this section - it's pretty Netflix + AWS specific.

There are a handful of other changes to this file that went in shortly after you started on the integration - you might want to grab the current `instanceDetails.html` file from the `amazon` module and use that as your template.